code/datums/components/transform_on_pickup.dm

```diff
@@ -0,0 +1,10 @@
+/// Resets the transformation on an item when it's picked up. Used for resizing things coming out of gang crates.
+/datum/component/transform_on_pickup
```

Comment:
This isn't what I'd expect this component to do from the name, maybe `reset_transform_on_pickup` would be clearer (if horribly verbose)